source/testnet/references/id-accounts.rst

```diff
@@ -74,13 +70,14 @@ on the provided documentation.
 
 The *Notabene (development)* flow allows the user to provide their own
 attributes and does no verification. This flow is available for testing
-purposes, and will not be supported on the `mainnet`_.
+purposes, and will not be supported on the :ref:`glossary-mainnet`.
 
 Creating an account
 ===================
 
-Once a user has an identity, they can use it to create a number of accounts. In
-contrast to obtaining an identity, opening an account is an `on-chain`_ action
+Once a user has an identity, they can use it to create a number of accounts, besides the
+`initial account`_ created by the identity provider upon creating the identity. In
```

Comment:
There does not seem to be a glossary entry for initial account.

Reply:
This entry is also added in another PR. (https://github.com/Concordium/concordium.github.io/pull/56)